.github/workflows/contributions.yml

```diff
@@ -0,0 +1,27 @@
+name: Our open source contributions
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '0 0 1 * *'
+  workflow_dispatch:
+jobs:
+  collect:
+    name: test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Fetch Docker image
+        run: docker pull jeroeng/ghcontributions:latest
+      - name: Do magic
+        run: |
+          curl -s -H "Accept: application/vnd.github+json" \
+            -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+            https://api.github.com/orgs/Enrise/members?per_page=100 \
+            | jq .[]?.login -r \
+            | while read name ; do
+                docker run --rm --volume $(pwd):/app contribs:latest python3 /app/generate_markdown.py $name -a
+              done \
+          >> $GITHUB_STEP_SUMMARY
```

Comment:
This shows the results (hopefully) in the summary of the job. Once that works the next steps should be:
- commit the contributions-list.md file
- merge that file into the readme